.gitignore

```diff
@@ -3,3 +3,6 @@
 node_modules/
 
 build/
+
+
+yarn.lock
```

Comment:
Shouldn't be ignored

Reply:
It looked like this project was originally made with npm and not yarn so I didn't want to add extra files but agree this should have been included from the start.

Reply:
It hasn't been included since Yarn didn't even exist when this project started :D